.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @mcfadden
```

Comment:
can we make this the platform team?

Reply:
This is a public gem, so CODEOWNERS can only reference public teams unfortunately.